packages/langium/src/serializer/json-serializer.ts

```diff
@@ -137,3 +154,27 @@ export class DefaultJsonSerializer implements JsonSerializer {
         return this.astNodeLocator.getAstNode(root, path.substring(1))!;
     }
 }
+
+function addAstNodeRegionWithAssignmentsTo(root: AstNodeWithTextRegions) {
```

Comment:
Suggestion: This function should be part of the serializer or exported so that users that override the service can use the function as well.